core/trino-main/src/test/java/io/trino/cost/TestJoinStatsRule.java

```diff
@@ -18,14 +18,18 @@
 import io.trino.spi.type.Type;
 import io.trino.sql.planner.Symbol;
 import io.trino.sql.planner.TypeProvider;
+import io.trino.sql.planner.iterative.rule.test.PlanBuilder;
 import io.trino.sql.planner.plan.JoinNode;
 import io.trino.sql.planner.plan.JoinNode.EquiJoinClause;
+import io.trino.sql.planner.plan.PlanNode;
 import io.trino.sql.tree.ComparisonExpression;
 import io.trino.sql.tree.LongLiteral;
 import org.testng.annotations.Test;
 
 import java.util.Optional;
+import java.util.function.Function;
 
+import static io.trino.SystemSessionProperties.JOIN_MULTI_CLAUSE_INDEPENDENCE_FACTOR;
```

Comment:
could we have some test cases for `independence_factor=1 and 0`?

Reply:
they're already there in `testStatsForInnerJoinWithTwoEquiClauses`